### DISCUSSION TITLE
Meeting to discuss alignment with CA meeting finder

### DISCUSSION THREAD

**Discussion by canewyork** [code4recovery/12-step-meeting-list#519] [General]
Hi Folks -

We have been developing a meeting finder for World Service CA for a few months now. When we found your meeting finder code base - we saw a tremendous amount of overlap.

The team is one dev in NY and other dev in Sweden. We have built a few things already for the fellowship around an admin tool and a few plugins.

We wanted to set up a call to introduce ourselves and hopefully join forces to help with your existing codebase.

Please let me know what day/time works for you.

Looking forward to having a conversation with you.

**Comment by tech2serve** [comment]
Good morning. I've sent you an email with possible call dates.
